requirements-optional.txt

```diff
@@ -40,5 +40,5 @@ wordcloud>=1.8.1
 textblob>=0.17.1
 
 # Parallel
-fugue>=0.6.6
+fugue~=0.6.6
```

Comment:
Isn't there a fix of the code to make it work with the latest version?

Reply:
We'll do that in a followup PR, this is to make CI pass